shub/login.py

```diff
@@ -1,21 +1,56 @@
-import os, re, click
-from shub.utils import get_key_netrc, NETRC_FILE
+import re
+import click
+import os
+from shub import auth
+from scrapy.utils.conf import get_config
```

Comment:
Maybe not yet, but soon enough. For example, if you want to play with the data, not deploy anything.

```
$ shub login
$ shub items 123/45/67 | grep 'product title' | python -mjson.tool
```